### DISCUSSION TITLE
cant set up pulse_time in my compiled firmware

### DISCUSSION THREAD

**Discussion by scxrrow** [arendst/Tasmota#18780] [Q&A] [answered]
i'm working on a dual relay where i set up PulseTime to 5 through the mqtt commands. Now i want to have it configured on my compiled firmware. but the problem is, i dont find how to do. 

thanks in advance for your help.

**Comment by sfromis** [comment]
`#define APP_PULSETIME          0                 // [PulseTime] Time in 0.1 Sec to turn off power for relay 1 (0 = disabled)`

**Reply by scxrrow** [reply]
ok thank you, i was using the CMND in the template but it wasnt working. 

Is it possible to configure it for the second relay with your solution ?

**Reply by sfromis** [reply]
Not that way, unless you swap what is relay 1 vs 2.

You also have the more general:
`#define USER_BACKLOG "<Any command separated by a semicolon (;)>"  // Add commands executed at firmware load or when command reset is executed`

**Reply by sfromis** [reply, marked as answer]
Using CMND in the template works *once only* when the template is applied, not somehow every time (not even every boot). Using `USER_BACKLOG` is similar, only takes effect after a "factory reset", not every boot.
